BitwardenShared/UI/Auth/Extensions/AuthRouter+Redirects.swift

```diff
@@ -43,6 +42,14 @@ extension AuthRouter {
             }
         }
 
+        do {
+            if let rehydratableTarget = try await services.rehydrationHelper.getSavedRehydratableTarget() {
+                return .completeWithRehydration(rehydratableTarget)
```

Comment:
🤔 Should this be behind the `if !isInAppExtension` check above? (Maybe the AppCoordinator changes prevent this route from being used in the extension, but might not hurt to also check it here?)

Reply:
Yep totally agree, it's safer to do it in there so just moved it, thanks 😄